Resources/Private/Fusion/Consent.fusion

```diff
@@ -2,8 +2,15 @@
 prototype(Sandstorm.CookiePunch:Consent) < prototype(Neos.Fusion:Component) {
     noCSS = false
 
+    activeServices = ${Array.filter(Configuration.setting("Sandstorm.CookiePunch.consent.services"), service => service.when == null || ConditionalRendering.checkCondition(service.when, site))}
```

Comment:
I'm not very happy about pulling the services out of the Sandstorm.CookiePunch:Config and have that detail here. It's the only way I found to prevent the rendering of the whole Consent when there is no service left

Reply:
Seems to be the best way we can come up with after discussing it in a code review